Cargo.toml

```diff
@@ -101,30 +101,30 @@ clvm-traits = { workspace = true, features = ["derive"] }
 clvmr = { workspace = true }
 
 [workspace.dependencies]
-chia-wallet-sdk = { version = "0.26.0", path = "." }
-chia-sdk-client = { version = "0.26.0", path = "./crates/chia-sdk-client" }
-chia-sdk-driver = { version = "0.26.0", path = "./crates/chia-sdk-driver" }
-chia-sdk-signer = { version = "0.26.0", path = "./crates/chia-sdk-signer" }
-chia-sdk-test = { version = "0.26.0", path = "./crates/chia-sdk-test" }
-chia-sdk-types = { version = "0.26.0", path = "./crates/chia-sdk-types" }
-chia-sdk-derive = { version = "0.26.0", path = "./crates/chia-sdk-types/derive" }
-chia-sdk-utils = { version = "0.26.0", path = "./crates/chia-sdk-utils" }
-chia-sdk-coinset = { version = "0.26.0", path = "./crates/chia-sdk-coinset" }
-chia-sdk-bindings = { version = "0.26.0", path = "./crates/chia-sdk-bindings" }
-bindy = { version = "0.26.0", path = "./crates/chia-sdk-bindings/bindy" }
-bindy-macro = { version = "0.26.0", path = "./crates/chia-sdk-bindings/bindy-macro" }
-chia = "0.25.0"
-chia-ssl = "0.25.0"
-chia-protocol = "0.25.0"
-chia-consensus = "0.25.0"
-chia-traits = "0.25.0"
-chia-bls = "0.25.0"
-chia-secp = "0.25.0"
-chia-puzzle-types = "0.25.0"
-clvm-traits = "0.25.0"
-clvm-utils = "0.25.0"
-chia-sha2 = "0.25.0"
-chia_streamable_macro = "0.25.0"
+chia-wallet-sdk = { version = "0.27.1", path = "." }
+chia-sdk-client = { version = "0.27.1", path = "./crates/chia-sdk-client" }
+chia-sdk-driver = { version = "0.27.1", path = "./crates/chia-sdk-driver" }
+chia-sdk-signer = { version = "0.27.1", path = "./crates/chia-sdk-signer" }
+chia-sdk-test = { version = "0.27.1", path = "./crates/chia-sdk-test" }
+chia-sdk-types = { version = "0.27.1", path = "./crates/chia-sdk-types" }
+chia-sdk-derive = { version = "0.27.1", path = "./crates/chia-sdk-types/derive" }
+chia-sdk-utils = { version = "0.27.1", path = "./crates/chia-sdk-utils" }
+chia-sdk-coinset = { version = "0.27.1", path = "./crates/chia-sdk-coinset" }
+chia-sdk-bindings = { version = "0.27.1", path = "./crates/chia-sdk-bindings" }
+bindy = { version = "0.27.1", path = "./crates/chia-sdk-bindings/bindy" }
+bindy-macro = { version = "0.27.1", path = "./crates/chia-sdk-bindings/bindy-macro" }
+chia = "0.26.0"
+chia-ssl = "0.26.0"
+chia-protocol = "0.26.0"
+chia-consensus = "0.26.0"
+chia-traits = "0.26.0"
+chia-bls = "0.26.0"
+chia-secp = "0.26.0"
+chia-puzzle-types = "0.26.0"
+clvm-traits = "0.26.0"
+clvm-utils = "0.26.0"
+chia-sha2 = "0.26.0"
+chia_streamable_macro = "0.26.0"
 chia-puzzles = "0.20.1"
```

Comment:
What determines which chia package here gets a version bump? looking at chia-puzzles.   I'm guessing that chia-puzzles is static and self contained so it can be left alone?